modules/search_api_solr_autocomplete/src/Plugin/search_api_autocomplete/suggester/BackendTrait.php

```diff
@@ -1,6 +1,6 @@
 <?php
```

Comment:
Same as for `Spellcheck` but consider this docs https://www.drupal.org/about/core/policies/core-change-policies/drupal-deprecation-policy#how-class